pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -888,20 +892,34 @@ func (u *UnschedulablePods) addOrUpdate(pInfo *framework.QueuedPodInfo) {
 	u.podInfoMap[podID] = pInfo
 }
 
-// Delete deletes a pod from the unschedulable podInfoMap.
-func (u *UnschedulablePods) delete(pInfo *framework.QueuedPodInfo) {
+// delete deletes a podInfo from the unschedulable podInfoMap.
+// The `gated` parameter is used to figure out which metric should be decreased.
+func (u *UnschedulablePods) delete(pInfo *framework.QueuedPodInfo, gated bool) {
```

Comment:
since we are not using anything from pInfo, lets only pass the pod instead

Reply:
done.